npm-convert.js

```diff
@@ -259,7 +259,7 @@ function convertToPackage(context, pkg, index) {
 		localPkg = {
 			name: pkg.name,
 			version: pkg.version,
-			fileUrl: pkg.fileUrl,
+			fileUrl: steal.relativeURI(context.loader.baseURL, pkg.fileUrl),
```

Comment:
system-npm technically supports SystemJS (at least older versions), so shouldn't depend on steal like this. I would make a function in npm-utils.js that is:

```js
relative: function(baseURL, url){
  return typeof steal !== "undefined" ? steal.relativeURI(baseURL, url) : url;
}
```

Basically only applying this function if we are running in Steal.